packages/block-library/src/post-excerpt/edit.js

```diff
@@ -2,16 +2,96 @@
  * WordPress dependencies
  */
 import { useEntityProp, useEntityId } from '@wordpress/core-data';
-import { PlainText } from '@wordpress/block-editor';
+import { useMemo } from '@wordpress/element';
+import { InspectorControls, RichText } from '@wordpress/block-editor';
+import { PanelBody, RangeControl, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
 
-function PostExcerptDisplay() {
+function usePostContentExcerpt( excerptLength ) {
+	const [ , , { raw: rawPostContent } ] = useEntityProp(
+		'postType',
+		'post',
+		'content'
+	);
+	return useMemo( () => {
+		if ( ! rawPostContent ) {
+			return '';
+		}
+		const excerptElement = document.createElement( 'div' );
```

Comment:
Is this the equivalent of the p element you use later in the PHP file? In other words, is this going to be used as a paragraph with inner text and a link, or is this a wrapper that will contain a p and something else? If the former, it should be a p element instead of a div, too.

Reply:
It's not rendered anywhere. It's just for parsing the HTML into a transformable text string.